aips/aip-116.md

```diff
@@ -114,27 +111,49 @@ The `AptosSignInInput` interface will be included in the **signing request** to
-| ----------------- | ------------------ | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| account.address   | string             | ✓         | The address of the user that is signing in.                                                                                                                                                                                                           |
-| account.publicKey | bytes              | ✓         | The public key of the user that is signing in.                                                                                                                                                                                                        |
-| input             | AptosSignInInput\* | ✓         | The AptosSignInInput that contains the fields used to construct the message. \* The following fields must be bound: `address`, `chainId`, `domain`, `uri`, and `version`. If the bound fields are not provided, they must be populated by the wallet. |
-| plainText         | string             | ✓         | The constructed message using the standardized message format.                                                                                                                                                                                        |
-| signingMessage    | bytes              | ✓         | The signing message that conforms to the domain separated signing algorithm.                                                                                                                                                                          |
-| signature         | bytes              | ✓         | The signature of the signing message.                                                                                                                                                                                                                 |
-| signatureType     | string             | ✓         | The key scheme used to sign the message (e.g. 'ed25519', 'single_key', 'multi_key', 'multi_ed25519', etc.).                                                                                                                                           |
+| Name              | Type             | Description                                                                                                        |
+| ----------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------ |
+| account.address   | string           | The address of the user that is signing in. This field MUST match the `address` field used to construct `message`. |
```

Comment:
It is very unclear what `message` is referring to here, especially since it is quoted with `` .